commandapi-core/src/main/java/dev/jorel/commandapi/AbstractCommandAPICommand.java

```diff
@@ -248,11 +292,41 @@ public void register() {
 
 	public Impl copy() {
 		Impl command = newConcreteCommandAPICommand(new CommandMetaData<>(this.meta));
-		command.args = new ArrayList<>(this.args);
+		command.arguments = new ArrayList<>(this.arguments);
 		command.subcommands = new ArrayList<>(this.subcommands);
 		command.isConverted = this.isConverted;
 		return command;
 	}
 
 	protected abstract Impl newConcreteCommandAPICommand(CommandMetaData<CommandSender> metaData);
+
+	private List<Argument[]> getArgumentsToRegister(Argument[] argumentsArray, List<Argument[]> argumentsToRegister) {
```

Comment:
Does this method need to take in the List that the arguments should be added to, then return that list?

I don't think there's any reason this wouldn't be an empty `ArrayList`, so it can probably just be created inside this method.

If you do pass in the List, you technically also don't need to return that List, because the outer function could already have a reference to it. Eg.
```java
List<?> list = new ArrayList<>();
modifyList(list);
print(list); // This list has been modified by the function
```